### PR TITLE
Replaced all old usages of String:: with their new equivalent

### PR DESCRIPTION
**Pull request by LKS90**
String::checkPlain -> SafeMarkUp::checkPlain and String::decodeEntitites -> Html::decodeEntities. Removed class declaration at the beginning of files. I did not see any additional classes that had to be replaced. All tests passed.

### CODE REVIEW

**Comment by kimpepper** [comment]
:+1:

File: src/Tests/PathautoUnitTest.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 namespace Drupal\pathauto\Tests;
 
+use Drupal\Component\Utility\SafeMarkup;
 use Drupal\Core\Language\Language;
 use Drupal\Component\Utility\String;
```
Comment (Berdir) [inline src/Tests/PathautoUnitTest.php]: There's still a use here for the old class.

File: src/Tests/PathautoUserWebTest.php [hunk 2]
```diff
@@ -73,8 +73,10 @@ function testUserOperations() {
     $view = Views::getView('user_admin_people');
     $view->initDisplay();
     $view->preview('page_1');
+
+
     foreach ($view->result as $key => $row) {
-      if ($row->users_field_data_name == $account->getUsername()) {
+      if ($view->field['name']->getValue($row) == $account->getUsername()) {
```
Comment (Berdir) [inline src/Tests/PathautoUserWebTest.php:79]: past might need a very similar fix for the severity stuff btw.

**Comment by LKS90** [comment]
Can you merge this one as well?

**Pull request merged**